js_modules/dagster-ui/packages/ui-core/src/assets/AssetEvents.tsx

```diff
@@ -124,8 +141,50 @@ export const AssetEvents = ({
     }
   };
 
+  const {cachedDefinition, definition} = useAssetDefinition(assetKey);
+
+  const def = definition ?? cachedDefinition;
+
+  const hasFilter =
```

Comment:
I wonder if this should go into `useAssetEventsFilters` or up generally in the file? Worried about this not being kept in sync with the filters

Reply:
ah yeah I think I could check `activeFiltersJsx.length` and that would be sufficient. I'll check.